Assets/Scripts/Health.cs

```diff
@@ -8,6 +8,10 @@ public class Health : MonoBehaviour {
 	public int maxHealth = 10;
 	private int currentHealth;
 
+    public float dropProbability; // number between 0.0 and 1.0
```

Comment:
Don't couple the pickup drops with the health component. **Remember** Health component should be able to work even without any drops. Make this its own component